### ISSUE TITLE
训练时候图片大小不固定问题的解决?

### ISSUE THREAD

**Issue by zHanami** [meteorshowers/RCF-pytorch#2]
When I train, I find the loss is not stable, in one epoch, it can convergence; but when it comes to another epoch, the loss become very big.

**Comment by meteorshowers** [comment]
yes, it is
